examples/spark/images/base/core-site.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+
+<configuration>
+  <property>
+    <name>fs.gs.impl</name>
+    <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem</value>
```

Comment:
@timstclair unless you have evidence otherwise, i'd expect this is just *adding* GoogleHadoopFileSystem, not making GHFS the only option

fyi, @wattsteve maintains https://wiki.apache.org/hadoop/HCFS, which suggests that are ~12 such filesystem implementations.

@zmerlynn imho,the question is if we should include hcfs plugins in the image or not. since realistically there are maybe 3 or 4 that are relevant and available (some are prioprietary and not distributable), i'm for including them in the image instead of providing separate images for each or making the image pluggable.